src/test/mds/TestMDSAuthCaps.cc

```diff
@@ -234,6 +242,8 @@ TEST(MDSAuthCaps, OutputParsed) {
     const char *output;
   };
   CapsTest test_values[] = {
+    {"",
+     "MDSAuthCaps[allow rw]"},
     {"allow",
```

Comment:
does this mean "empty string implies rw"?

Reply:
Yes, but I don't like it.